pkgs/applications/networking/remote/remmina/default.nix

```diff
@@ -29,15 +30,16 @@ stdenv.mkDerivation rec {
     freerdp libssh libgcrypt gnutls
     pcre2 libdbusmenu-gtk3 libappindicator-gtk3
     libvncserver libpthreadstubs libXdmcp libxkbcommon
-    libsecret libsoup spice-protocol spice-gtk libepoxy at-spi2-core
+    libsoup spice-protocol spice-gtk libepoxy at-spi2-core
     openssl gnome.adwaita-icon-theme json-glib libsodium webkitgtk
     harfbuzz
-  ];
+  ] ++ optionals withLibsecret [ libsecret ];
 
   cmakeFlags = [
     "-DWITH_VTE=OFF"
     "-DWITH_TELEPATHY=OFF"
     "-DWITH_AVAHI=OFF"
+    "-DWITH_LIBSECRET=${if withLibsecret then "ON" else "OFF"}"
```

Comment:
I suppose this pattern should probably be put into `lib/strings.nix` or such, because it appears in many CMake projects.

Reply:
Yes, it would be nice if you could do it and then treewide replace occurences.

Reply:
I could do it, but I'm currently somewhat busy, and noticed that it is really widespread in the tree; as such, I won't do it as part of this PR, because (1) I currently don't really have enough time to do it and (2) it is off-topic to this PR and would just cause it to affect dozens of additional packages, making it even less likely that this PR might be merged soon (because it would become too big to be reviewed in a reasonable time frame).
I might do it later, tho, and probably depending on the fact that this PR gets merged because routinely rebasing multiple treewide PRs is just too much work for me; and because this PR introduces another occurrence. Also, it might take a while until I have enough time to pull that off.

Reply:
```console
[zseri@orca:~/devel/nixpkgs]$ rg -F 'then "ON" else "OFF"' | wc -l
127
```